### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).

It also includes following components:

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/Exceptions](https://hosted.weblate.org/projects/kimai/exceptions/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Invoice-Numbers](https://hosted.weblate.org/projects/kimai/invoice-numbers/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Actions (Dropdowns and Toolbars)](https://hosted.weblate.org/projects/kimai/actions/)

* [Kimai - Time tracking/About](https://hosted.weblate.org/projects/kimai/about/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: translations/invoice-renderer.en.xlf [hunk 1]
```diff
@@ -20,20 +20,20 @@
         <target>HTML (for printing via Browser)</target>
       </trans-unit>
       <trans-unit id="4NNx9U." resname="Docx">
-        <source>Docx</source>
-        <target>Microsoft Word (Docx)</target>
+        <source>Microsoft Word (DOCX)</source>
```
Comment (kevinpapst) [inline translations/invoice-renderer.en.xlf:23]: VERIFY! Probably reject...

File: translations/invoice-renderer.nb-NO.xlf [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        <source>Microsoft Excel (XLS)</source>
+        <target>Microsoft Excel (XLS)</target>
+      </trans-unit>
+      <trans-unit id="QqvUWFt" resname="Ods">
+        <source>Ods</source>
+        <target state="needs-translation">LibreOffice (regneark)</target>
+      </trans-unit>
+      <trans-unit id="sjwwrr8" resname="Xlsx">
+        <source>Microsoft Excel (XLSX)</source>
+        <target>Microsoft</target>
```
Comment (kevinpapst) [inline translations/invoice-renderer.nb-NO.xlf:36]: Excel

**Comment by kevinpapst** [comment]
@comradekingu thank you very much for your contributions 👍 I'd like to ask for one favor though: can you please create a new language only if you have plans to translate it (soon). The entire translation process here fails if there are empty keys/files and then I have to apply manual changes for every missing key ... as the framework unfortunately cannot work with these empty keys.

File: translations/invoice-renderer.en.xlf (hunk 1 above)
Comment (comradekingu) [reply translations/invoice-renderer.en.xlf:23]: It is either DOCX or .docx https://docs.fileformat.com/word-processing/docx/

**Comment by comradekingu** [comment]
@kevinpapst nb_NO has full coverage now. It didn't make it into this PR though.
There are provisions for populating empty strings in Weblate, and also I would suggest the plugin that adds all components for any started language.

**Comment by kevinpapst** [comment]
Super cool, that was quick, thanks @comradekingu 👍 
I had the issue often in the past that users created a language but then did not translate anything.

Can you please have a look here, there are two errors in new translations:
https://github.com/kevinpapst/kimai2/runs/5352827095?check_suite_focus=true
I hope the errors are clear understandable? 

Thanks for the recommendation. I am really not a Weblate user, so I am happy about every hint to make the experience better for you. I activated this plugin: https://docs.weblate.org/en/latest/admin/addons.html#addon-weblate-consistency-languages - is that the one you mentioned?

**Comment by kevinpapst** [comment]
Wow, that plugin fucks up the entire repository. I deactivated it and locked the repository, but it keeps on posting...

**Comment by kevinpapst** [comment]
I activated `nb_NO` as locale, will be released with the next minor release 1.18.1.
Now I will try to activate the weblate plugin again.

Thanks again @comradekingu 👍

**Pull request merged**